app/views/groups/index.html.erb

```diff
@@ -0,0 +1,35 @@
+<div class="container">
+  <div class="d-flex justify-content-between">
+    <h3>All Categories</h3>
+    <%= link_to 'Sign out', destroy_user_session_path, method: :delete %>
+  </div>
+
+  <div class="categories">
+    <% @groups.each_with_index do |cat, index|%>
+      <%= link_to group_path(cat) do %>
+        <div class="category card mb-3">
+          <div class="row g-0">
+             <div class="col-md-4">
+             <%= image_tag(cat.icon, class: "img-fluid", alt: cat.name, style: "height: 100px; width:100px;") if cat.icon.attached? %>
+        </div>
+            <div class="col-md-8">
+              <div class="card-body text-dark">
+                <div class="d-flex justify-content-between">
+                  <h5 class="card-title"><%= cat.name %></h5>
+                <p class="card-text">Amount: $<%= @totals[index] || 0 %></p>
+                </div>
```

Comment:
- [ ] This section is meant to show the total amount of all the transactions that belong to that category, not the amount.
- For example, if you have 4 transaction, it shows 4, not the sum of the transactions

### Change view:
![image](https://github.com/arnoldnekemiah/Budget-app/assets/50990925/0e7f2414-7040-4e83-a72d-794f911ffc7a)